docs/catalogue/cat_network-data-manager.md

```diff
@@ -43,48 +48,41 @@ section [Upload metadata](cat_network-data-manager.md#upload-metadata) to the Da
 
 <sup>*Figure 1. Tables in a Network’s staging area in the Data Catalogue.*</sup>
 
-#### *Releases* sheet
+#### *TargetDataDictionaries* sheet
 
-Versioning of the network's data model is defined in the *Releases* sheet. Columns with an asterisk (\*) after their
-name are mandatory.
+Versioning of the network's data model is defined in the *TargetDataDictionaries* sheet. Columns with an asterisk (\*) 
+after their name are mandatory.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
-| resource \* | Name of the network. | e.g LifeCycle, LONGITOOLS or ATHLETE |
-| version \* | Version of the data release. | e.g. 1.1 |
-
-<sup>*Table 1. Description of the columns that can be filled out for Releases. \* = mandatory*</sup>
+| resource \* | Name of the network's model. | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
+| version \* | Version of the data dictionary. | e.g. 1.1 |
 
-![Figure 2. Example of Releases filled out in Excel.](../img/cat_target-release-sheet.png ':size=150')
-
-<sup>*Figure 2. Example of Releases filled out in Excel.*</sup>
+<sup>*Table 1. Description of the columns that can be filled out for TargetDataDictionaries. \* = mandatory*</sup>
 
 #### *TargetTables* sheet
 
 The cohort tables are defined in the *SourceTables* sheet. Columns with an asterisk (\*) after their name are mandatory.
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
-| release.resource \* | Name of the network. | e.g LifeCycle, LONGITOOLS or ATHLETE |
-| release.version \* | Release version this table is part of. | e.g. 1.0.1 |
+| dataDictionary.resource \* | Name of the model. | e.g LifeCycle_CDM, LongITools_CDM or ATHLETE_CDM |
+| dataDictionary.version \* | Data dictionary version this table is part of. | e.g. 1.0.1 |
 | name \* | Unique table name. | |
 | label | Table label. | |
 | description | Table description. | |
 
 <sup>*Table 2. Description of the columns that can be filled out for TargetTables. \* = mandatory*</sup>
 
-![Figure 3. Example of TargetTables filled out in Excel.](../img/cat_target-tables-sheet.png ':size=600')
-
-<sup>*Figure 3. Example of TargetTables filled out in Excel.*</sup>
-
 #### *TargetVariables* sheet
 
```

Comment:
NetworkDictionary template and NetworkDictionary_testCDM contain extra columns: references.table, references.name that are not listed in the table below

Reply:
these are probably never used, I will delete them from the template